### PR TITLE
[ISSUE #42] fix the problem HealthExcutor don't doCheck asynchronously

### PR DESCRIPTION
**Pull request by Lambert-Rao**
Besides, I add an integration test for HealthService

### CODE REVIEW

File: eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/health/CheckResultCache.java [hunk 1]
```diff
@@ -41,8 +41,10 @@ public void update(String type, Long typeId, HealthCheckStatus status, String re
             cacheMap.put(type, subMap);
         }
         CheckResult oldResult = subMap.get(typeId);
-        String oldDesc = Objects.isNull(oldResult.getResultDesc()) ? "" : oldResult.getResultDesc() + "\n";
-        CheckResult result = new CheckResult(status, oldDesc + resultDesc, LocalDateTime.now(),
+        String oldDesc = Objects.isNull(oldResult.getResultDesc()) || oldResult.getResultDesc().isEmpty() ? "" : oldResult.getResultDesc() + "\n";
```
Comment (Pil0tXia) [inline eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/health/CheckResultCache.java:44]: Why `oldDesc` should have a `\n` at the end?

Comment (Lambert-Rao) [reply eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/health/CheckResultCache.java:44]: ` newdesc` is added after `oldDesc` , just to distinguish

Comment (Pil0tXia) [reply eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/health/CheckResultCache.java:44]: You mean like this?

```
oldDesc
newdesc
```

By the way, `Objects.isNull` is intended for use within Java 8 lambda filtering. Just keep the code simple using `whatever != null` will be better.

Comment (Lambert-Rao) [reply eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/health/CheckResultCache.java:44]:
```
oldDesc
newdesc
```
that's what I mean

Comment (Pil0tXia) [reply eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/health/CheckResultCache.java:44]: Then such writting will be better:

```
String whatever = oldDesc + Constants.NEW_LINE_ENDING + newDesc;
```

What do you think?

Comment (Lambert-Rao) [reply eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/health/CheckResultCache.java:44]: okay

Comment (Pil0tXia) [reply eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/health/CheckResultCache.java]: Well, using a constant is meant to split `oldDesc` and `newDesc` strings. So the following is better:

```
String description = oldDesc + Constants.NEW_LINE_ENDING + resultDesc;
```

Also, a constant can be referenced by its class name to indicate its usage field, so a `HEALTH_` prefix is unnecessary.

Comment (Lambert-Rao) [reply eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/health/CheckResultCache.java]: if `oldDesc` is an empty string `""`, this add an `NEW_LINE_ENDING` is the beginning of the `description`.

Comment (Pil0tXia) [reply eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/health/CheckResultCache.java]: if-else is acceptable~

**Review by Pil0tXia — APPROVED** [review]

**Pull request merged**